### PR TITLE
Add changelog entry for Flux v1.11.0

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,101 @@
...
+This release comes with experimental garbage collection and Git commit signing:
+
+1. Experimental garbage collection of cluster resources. When providing the
+   `--sync-garbage-collection` flag, namespace-scoped cluster resources no
+   longer existing in Git will be removed. Read the
+   [garbage collection documentation](site/garbagecollection.md) for further
+   details.
+
+2. [GPG](https://en.wikipedia.org/wiki/GNU_Privacy_Guard)
+   [Git commit signing](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Sltkeyidgt).
```
Comment (hiddeco) [inline CHANGELOG.md]:
```suggestion
   [Git commit signing](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Sltkeyidgt),
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,101 @@
...
+- Apply user defined Git timeout on all operations
+  [weaveworks/flux#1767][#1767]
+
+### Maintenance and documentation
+
+- Bump Alpine version from v3.6 to v3.9
+  [weaveworks/flux#1801][#1801]
+- Increase memcached memory defaults
+  [weaveworks/flux#1780][#1780]
+- update developing docs to remind to `make test`
```
Comment (hiddeco) [inline CHANGELOG.md]:
```suggestion
- Update developing docs to remind to `make test`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,101 @@
...
+   `--sync-garbage-collection` flag, namespace-scoped cluster resources no
+   longer existing in Git will be removed. Read the
+   [garbage collection documentation](site/garbagecollection.md) for further
+   details.
+
+2. [GPG](https://en.wikipedia.org/wiki/GNU_Privacy_Guard)
+   [Git commit signing](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Sltkeyidgt).
+   when providing `--git-signing-key` flag. GPG keys can be imported with
+   `--git-gpg-key-import`. By default Flux will import to and use the keys
+   in `~/.gnupg`. This path can be overriden by setting the `GNUPGHOME` environment
```
Comment (squaremo) [inline CHANGELOG.md]: "overridden"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,101 @@
...
+
+### Maintenance and documentation
+
+- Bump Alpine version from v3.6 to v3.9
+  [weaveworks/flux#1801][#1801]
+- Increase memcached memory defaults
+  [weaveworks/flux#1780][#1780]
+- update developing docs to remind to `make test`
+  [weaveworks/flux#1796][#1796]
+- fix Github link
```
Comment (hiddeco) [inline CHANGELOG.md]:
```suggestion
- Fix Github link
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,101 @@
...
+  [weaveworks/flux#1771][#1771]
+- Increase timeouts in daemon_test.go
+  [weaveworks/flux#1779][#1779]
+- Rename resource method Policy() to Policies()
+  [weaveworks/flux#1775][#1775]
+- Improve testing in local environments other than linux-amd64
+  [weaveworks/flux#1765][#1765]
+- Re-flow sections to order by importance
+  [weaveworks/flux#1754][#1754]
+- document flux-dev mailing list
```
Comment (hiddeco) [inline CHANGELOG.md]:
```suggestion
- Document flux-dev mailing list
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,101 @@
...
+   when providing `--git-signing-key` flag. GPG keys can be imported with
+   `--git-gpg-key-import`. By default Flux will import to and use the keys
+   in `~/.gnupg`. This path can be overriden by setting the `GNUPGHOME` environment 
+   variable.
+
+   Commit signature verification is in the works and wil be released shortly.
+
+### Fixes
+
+- Wait for shutdown before returning from main()
```
Comment (hiddeco) [inline CHANGELOG.md]:
```suggestion
- Wait for shutdown before returning from `main()`
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,101 @@
...
+   details.
+
+2. [GPG](https://en.wikipedia.org/wiki/GNU_Privacy_Guard)
+   [Git commit signing](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Sltkeyidgt).
+   when providing `--git-signing-key` flag. GPG keys can be imported with
+   `--git-gpg-key-import`. By default Flux will import to and use the keys
+   in `~/.gnupg`. This path can be overriden by setting the `GNUPGHOME` environment 
+   variable.
+
+   Commit signature verification is in the works and wil be released shortly.
```
Comment (squaremo) [inline CHANGELOG.md]: "will"

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,101 @@
...
+
+### Fixes
+
+- Wait for shutdown before returning from main()
+  [weaveworks/flux#1789][#1789]
+- Let `fluxctl list-images` adhere to namespace filter
```
Comment (squaremo) [inline CHANGELOG.md]: s/Let/Make/

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,101 @@
...
+  [weaveworks/flux#1763][#1763]
+- Take ignore policy into account when working with automated resources
+  [weaveworks/flux#1749][#1749]
+- Run signal listener in a goroutine instead of deferring.
```
Comment (squaremo) [inline CHANGELOG.md]: This one is Helm op code; sorry, I didn't spot that before

**Comment by 2opremio** [comment]
All addressed, PTAL

File: CHANGELOG.md (hunk 4 above)
Comment (hiddeco) [inline CHANGELOG.md]: You missed this one

File: CHANGELOG.md (hunk 1 above)
Comment (hiddeco) [inline CHANGELOG.md]: It should either be

> GPG Git commit signing. When providing [...].

or

> GPG Git commit signing, when providing [...].

where the latter makes more sense to me.

**Comment by hiddeco** [comment]
@2opremio see comments and comment I unresolved.

**Comment by 2opremio** [comment]
> see comments and comment I unresolved.

@hiddeco I had incorrectly pushed the fixes to `2opremio/flux` and not `weaveworks/flux` (I am not used to work with upstream branches directly), sorry about that. Now it should be fixed for real.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,6 +1,99 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.11.0 (2019-03-13)
+
+This release comes with experimental garbage collection and Git commit signing:
+
+1. Experimental garbage collection of cluster resources. When providing the
+   `--sync-garbage-collection` flag, namespace-scoped cluster resources no
```
Comment (squaremo) [inline CHANGELOG.md]: I didn't notice this bit -- it is all resources, not just those in namespaces (unless you know something I don't? :-)

Comment (2opremio) [reply CHANGELOG.md]: I recall reading a test which verified this. I'm probably wrong, I will fix this tomorrow.

**Pull request merged**